### PR TITLE
:sparkles:  fix: add github.com as default for owner/repo parameter 

### PR DESCRIPTION
**Pull request by nanikjava**
* **Please check if the PR fulfills these requirements**
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] PR title follows the guidelines defined in  https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

New feature outlined #780 

* **What is the current behavior?** (You can also link to an open issue here)

Currently, it does not accept `repo/owner` format

* **What is the new behavior (if this is a feature change)?**

New behaviour will accept `repo/owner` format and using `github.com` as the default domain.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

No

* **Other information**:

### CODE REVIEW

**Review by laurentsimon — APPROVED** [review]
LGTM. Make sure to fix the failing checks and rebase your branch as it will prevent merging.
You can rename the magic values (2 and 3) and the failing check should then pass.

**Comment by nanikjava** [comment]
> LGTM. Make sure to fix the failing checks and rebase your branch as it will prevent merging.
> You can rename the magic values (2 and 3) and the failing check should then pass.

Thanks @laurentsimon , PR updated.

File: repos/repo_url.go [hunk 1]
```diff
@@ -56,12 +56,28 @@ func (r *RepoURL) String() string {
...
 func (r *RepoURL) Set(s string) error {
+	var t string
+
+	const two = 2
+	const three = 3
+
+	c := strings.Split(s, "/")
+
+	switch l := len(c); {
+	// this will takes care for repo/owner format, by default use github.com
```
Comment (laurentsimon) [inline repos/repo_url.go:67]: nit: start senetence with capitale letter and end with fulldot. Some checks may fail.
Also fix https://github.com/ossf/scorecard/pull/872/checks?check_run_id=3371804644

**Review by laurentsimon — APPROVED** [review]
I cannot merge until all the pre-submit checks succeed and your branch is rebased.

File: repos/repo_url.go (hunk 1 above)
Comment (nanikjava) [reply repos/repo_url.go]: Done.

**Comment by nanikjava** [comment]
> I cannot merge until all the pre-submit checks succeed and your branch is rebased.

The dilemma of different timezone :)

**Review by laurentsimon — APPROVED** [review]

**Pull request merged**